packages/vaadin-upload/src/vaadin-upload.js

```diff
@@ -333,45 +338,50 @@ class UploadElement extends ElementMixin(ThemableMixin(PolymerElement)) {
 
       {
         dropFiles: {
-        one: 'Drop file here',
-        many: 'Drop files here'
+          one: 'Drop file here',
+          many: 'Drop files here'
         },
         addFiles: {
-        one: 'Select File...',
-        many: 'Upload Files...'
+          one: 'Select File...',
+          many: 'Upload Files...'
         },
         cancel: 'Cancel',
         error: {
-        tooManyFiles: 'Too Many Files.',
-        fileIsTooBig: 'File is Too Big.',
-        incorrectFileType: 'Incorrect File Type.'
+          tooManyFiles: 'Too Many Files.',
+          fileIsTooBig: 'File is Too Big.',
+          incorrectFileType: 'Incorrect File Type.'
         },
         uploading: {
-        status: {
-          connecting: 'Connecting...',
-          stalled: 'Stalled',
-          processing: 'Processing File...',
-          held: 'Queued'
-        },
-        remainingTime: {
-          prefix: 'remaining time: ',
-          unknown: 'unknown remaining time'
+          status: {
+            connecting: 'Connecting...',
+            stalled: 'Stalled',
+            processing: 'Processing File...',
+            held: 'Queued'
+          },
+          remainingTime: {
+            prefix: 'remaining time: ',
+            unknown: 'unknown remaining time'
+          },
+          error: {
+            serverUnavailable: 'Server Unavailable',
+            unexpectedServerError: 'Unexpected Server Error',
+            forbidden: 'Forbidden'
+          }
         },
-        error: {
-          serverUnavailable: 'Server Unavailable',
-          unexpectedServerError: 'Unexpected Server Error',
-          forbidden: 'Forbidden'
-        }
+        file: {
+          retry: 'Retry',
+          start: 'Start',
+          remove: 'Remove'
         },
         units: {
-        size: ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'],
-        sizeBase: 1000
+          size: ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'],
+          sizeBase: 1000
         },
         formatSize: function(bytes) {
-        // returns the size followed by the best suitable unit
+          // returns the size followed by the best suitable unit
         },
         formatTime: function(seconds, [secs, mins, hours]) {
-        // returns a 'HH:MM:SS' string
+          // returns a 'HH:MM:SS' string
         }
       }
 
```

Comment:
Let's make this JSDoc comment use proper indentation with `*` characters, and add backticks to separate JS example, as suggested. Note, the lines below with `@type` and `@default` are misaligned, too.

Also, please update `.d.ts` file accordingly.

```suggestion
       *
       * ```
       * {
       *   dropFiles: {
       *     one: 'Drop file here',
       *     many: 'Drop files here'
       *   },
       *   addFiles: {
       *     one: 'Select File...',
       *     many: 'Upload Files...'
       *   },
       *   cancel: 'Cancel',
       *   error: {
       *     tooManyFiles: 'Too Many Files.',
       *     fileIsTooBig: 'File is Too Big.',
       *     incorrectFileType: 'Incorrect File Type.'
       *   },
       *   uploading: {
       *     status: {
       *       connecting: 'Connecting...',
       *       stalled: 'Stalled',
       *       processing: 'Processing File...',
       *       held: 'Queued'
       *     },
       *     remainingTime: {
       *       prefix: 'remaining time: ',
       *       unknown: 'unknown remaining time'
       *     },
       *     error: {
       *       serverUnavailable: 'Server Unavailable',
       *       unexpectedServerError: 'Unexpected Server Error',
       *       forbidden: 'Forbidden'
       *     }
       *   },
       *   file: {
       *     retry: 'Retry',
       *     start: 'Start',
       *     remove: 'Remove'
       *   },
       *   units: {
       *     size: ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'],
       *     sizeBase: 1000
       *   },
       *   formatSize: function(bytes) {
       *     // returns the size followed by the best suitable unit
       *   },
       *   formatTime: function(seconds, [secs, mins, hours]) {
       *     // returns a 'HH:MM:SS' string
       *   }
       * }
       * ```
       *
```